runWPT.sh

```diff
@@ -87,6 +87,9 @@ else
 fi
 
 TEST="${*: -1}"
+if [ -z "${TEST}" ]; then
```

Comment:
let's rewrite it as a node script :)

Reply:
It would be better indeed, as it would allow us to run it on Windows and finally enable Windows in CI.

Reply:
Although it's quite odd: This works on my machine but not in CI